core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -424,6 +438,10 @@ private void installConfig(PluginInfo info, Path tmpConfigDir, Path destConfigDi
         // create the plugin's config dir "if necessary"
         Files.createDirectories(destConfigDir);
 
+        final PosixFileAttributes destConfigDirAttributes =
+                Files.getFileAttributeView(destConfigDir.getParent(), PosixFileAttributeView.class).readAttributes();
+        setOwnerGroup(destConfigDir, destConfigDirAttributes);
```

Comment:
Can you explain the reason for this? It was explicitly removed because we should not be mucking with permissions. If the umask is set wrong, the onus is on the user.